### PR TITLE
add v1.0.0-pre2 changelog

### PR DESCRIPTION
**Pull request by nvkelso**
- adds initial `v1.0.0-pre2` **changelog** (culled from issues and PRs)
- add `v0.10.5` **changelog** (taken from release notes)
- corrects `v1.0.0-pre1` **dependencies** (oops)

### CODE REVIEW

**Comment by nvkelso** [comment]
Looks like CirlceCI failure is related to upstream OpenStreetMap data changes?

```
File "integration-test/502-water-boundaries-slow.py", line 48, in <module>
    "boundaries, but found %d boundaries." % num_boundaries
  File "/opt/circleci/python/2.7.10/lib/python2.7/contextlib.py", line 35, in __exit__
    self.gen.throw(type, value, traceback)
  File "./scripts/../integration-test.py", line 184, in features_in_tile_layer
    yield features
  File "integration-test/502-water-boundaries-slow.py", line 44, in <module>
    "test tile, but found only %d" % num_polygons
Exception: Tile 'http://localhost:34614/water/18/74987/100321.json': Expected at least 2 polygons in water boundary test tile, but found only 1
```

**Comment by nvkelso** [comment]
This didn't make it in last week:

`- **landuse** layer: Low- and mid-zoom landuse polygons are now merged within the same `kind` values (but tiered into several scale_rank sizes) to significantly reduce file size. Some properties, like `name`, `id`, `sport`, `religion`, and `surface` are dropped, and the `area` is recalculated for new combo polygons. ([#583](https://github.com/tilezen/vector-datasource/issues/583))`

Removed reference in c011be3e59e592380beca79c1e0d9c876b0c09e8.

**Comment by nvkelso** [comment]
Updates in https://github.com/tilezen/vector-datasource/pull/1023/commits/23572e5045de64697dcecfcf3367807639eb5da3 are ready to merge as soon as Circle gives green all clear:

* sort each section A-Z by layer name
* clarify release date
* add database note about min_zoom INT to REAL change

**Comment by nvkelso** [comment]
@iandees Easiest way to preview markdown file in this PR is by following the View file link in the Files Changed tab:

- https://github.com/tilezen/vector-datasource/blob/23572e5045de64697dcecfcf3367807639eb5da3/CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,130 @@
...
+
+- **pois** layer: Add art `gallery` features. ([#990](https://github.com/tilezen/vector-datasource/issues/990))
+
+- **pois** layer: Add `kind_detail` property sourced from `sport` for `pitch` features and sourced from `cuisine` for `biergarten`, `pub`, `bar`, `restaurant`, `fast_food`, `cafe` kinds (removing the `cuisine` property). ([#719](https://github.com/tilezen/vector-datasource/issues/719))
+
+- **roads** layer: To support highway shields a new `shield_text` property has been added, `network` values have been normalized (and bicycle networks are now excluded). An example: for "US 101" we now store `network` of  **US:US** and `shield_text` of **101**. Multiple shields are supported via optional `all_networks` and `all_shield_texts` lists (which work in GeoJSON and TopoJSON but not MVT formats, follow [mapbox-vector-tile/#64](https://github.com/tilezen/mapbox-vector-tile/issues/64] for a fix). The `ref` property remains available but is less useful for shield construction. ([#192](https://github.com/tilezen/vector-datasource/issues/192) and [896](https://github.com/tilezen/vector-datasource/issues/896))
```
Comment (iandees) [inline CHANGELOG.md]: `896` should be `#896`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,130 @@
...
+- **documentation**: Update **buildings** layer documentation to reflect new `kind` and `kind_detail` values and `building_part` changes. ([#933](https://github.com/tilezen/vector-datasource/issues/933)) and related ([#842](https://github.com/tilezen/vector-datasource/issues/842))
+
+- **documentation**: Update **roads** layer documentation to reflect new `kind` and `kind_detail` values, and remove erroneous reference to `exit`. ([#936](https://github.com/tilezen/vector-datasource/issues/936))
+
+- **contributing**: Adding a step to create a `test_config.yaml` file. ([#1001](https://github.com/tilezen/vector-datasource/pull/1001))
+
+
+  #### INTERNAL CHANGES (v1.0.0-pre2)
+
+- **performance**: Move label centroid calculation to database to reduce network pressure on the database (post processing transform needed to much geometry) in **landuse**, **water**, **earth**, and **buildings** layers. ([#965](https://github.com/tilezen/vector-datasource/issues/965))
```
Comment (iandees) [inline CHANGELOG.md]: `to` should be `too`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,130 @@
...
+
+- **performance**: Add pois indexes for OpenStreetMap polygon and point tables to improve query time. ([#983](https://github.com/tilezen/vector-datasource/issues/983))
+
+- **database**: Convert Postgres PostGIS database to `h-store` all. ([#876](https://github.com/tilezen/vector-datasource/issues/876))
```
Comment (iandees) [inline CHANGELOG.md]: The wording should probably be "database to utilize osm2pgsql's `--hstore-all` option"

**Comment by iandees** [comment]
Done looking. Left a few minor comments.

File: CHANGELOG.md (hunk 3 above)
Comment (nvkelso) [inline CHANGELOG.md]: Fixed in fb7c44c0d7722c68a2125aceaf09dadb49204177.

File: CHANGELOG.md (hunk 2 above)
Comment (nvkelso) [inline CHANGELOG.md]: Fixed in fb7c44c0d7722c68a2125aceaf09dadb49204177.

File: CHANGELOG.md (hunk 1 above)
Comment (nvkelso) [inline CHANGELOG.md]: Fixed in fb7c44c0d7722c68a2125aceaf09dadb49204177.

**Comment by nvkelso** [comment]
Merging!

**Pull request merged**